mission/config/subconfigs/vehicle_respawn_info.hpp

```diff
@@ -1,491 +1,1606 @@
+	};
+	class vn_b_army_static_m1919a4_high {
+		tags[] = {"m1919","armed","firesupport","static","machinegun"};
+	};
+	class vn_b_army_static_m1919a4_low {
+		tags[] = {"m1919","armed","firesupport","static","machinegun"};
+	};
+	class vn_b_army_static_m1919a6 {
+		tags[] = {"m1919","armed","firesupport","static","machinegun"};
+	};
+	class vn_b_army_static_mortar_m2 {
+		tags[] = {"m2","armed","firesupport","static","artillery","mortar"};
+	};
+	class vn_b_army_static_mortar_m29 {
+		tags[] = {"m29","armed","firesupport","static","artillery","mortar"};
+	};
+	class vn_b_army_static_m2_high {
+		tags[] = {"m2hb","armed","firesupport","static","machinegun"};
+	};
+	class vn_b_army_static_m2_scoped_high {
+		tags[] = {"m2hb","armed","firesupport","static","machinegun"};
+	};
+	class vn_b_army_static_m2_low {
+		tags[] = {"m2hb","armed","firesupport","static","machinegun"};
+	};
+	class vn_b_army_static_m2_scoped_low {
+		tags[] = {"m2hb","armed","firesupport","static","machinegun"};
+	};
+	class vn_b_army_static_m40a1rr {
+		tags[] = {"m40","armed","firesupport","static","antitank","nodisassemble"};
+	};
+	class vn_b_army_static_m45 {
+		tags[] = {"m45","armed","firesupport","static","antiair","nodisassemble"};
+	};
+	class vn_b_army_static_m60_high {
+		tags[] = {"m60","armed","firesupport","static","machinegun"};
+	};
+	class vn_b_army_static_m60_low {
+		tags[] = {"m60","armed","firesupport","static","machinegun"};
+	};
+	class vn_b_army_static_mk18 {
+		tags[] = {"m60","armed","firesupport","static","machinegun"};
+	};
+
+	//Boat
+	class vn_b_boat_12_01 {
+		tags[] = {"naval","pbr"};
+	};	
+	class vn_b_boat_12_02 {
+		tags[] = {"naval","pbr"};
+	};	
+	class vn_b_boat_12_03 {
+		tags[] = {"naval","pbr"};
+	};	
+	class vn_b_boat_12_04 {
+		tags[] = {"naval","pbr"};
+	};	
+	class vn_b_boat_13_01 {
+		tags[] = {"naval","pbr"};
+	};
+	class vn_b_boat_13_02 {
+		tags[] = {"naval","pbr"};
+	};
+	class vn_b_boat_13_03 {
+		tags[] = {"naval","pbr"};
+	};	
+	class vn_b_boat_13_04 {
+		tags[] = {"naval","pbr"};
+	};	
+	class vn_b_boat_05_01 {
+		tags[] = {"naval","nasty"};
+	};	
+	class vn_b_boat_05_02 {
+		tags[] = {"naval","nasty"};
+	};	
+	class vn_b_boat_06_01 {
+		tags[] = {"naval","nasty"};
+	};
+	class vn_b_boat_06_02 {
+		tags[] = {"naval","nasty"};
+	};
+	class vn_b_boat_09_01 {
+		tags[] = {"naval","stab"};
+	};	
+	class vn_b_boat_10_01 {
+		tags[] = {"naval","stab"};
+	};
+	class vn_b_boat_11_01 {
+		tags[] = {"naval","stab"};
+	};
+	class B_Boat_Transport_01_F {
+		tags[] = {"naval","RHIB"};
+	};
+
+	//DAC CONG
+	//Armor
+	class vn_o_armor_type63_01 {
+		tags[] = {"t63","pavn","armor"};
+	};
+	class vn_o_armor_m41_01 {
+		tags[] = {"m41","pavn","armor"};
+	};
+	class vn_o_armor_pt76a_01 {
+		tags[] = {"pt76","pavn","armor"};
+	};
+	class vn_o_armor_pt76b_01 {
+		tags[] = {"pt76","pavn","armor"};
+	};
+	class vn_o_armor_t54b_01 {
+		tags[] = {"t54","pavn","armor"};
+	};
+	class vn_o_armor_ot54b_01 {
+		tags[] = {"t54","pavn","armor"};
+	};
```

Comment:
Vehicle class does not exist. ...... see other comments about the issue.

Reply:
resolved